bundle/terranova/processor.go

```diff
@@ -0,0 +1,110 @@
+package terranova
+
+import (
+	"fmt"
+	"slices"
+
+	"github.com/databricks/cli/libs/dyn"
+	"github.com/databricks/cli/libs/dyn/merge"
+)
+
+type Move struct {
+	Fields []string
+	Target string
+}
+
+// Example:
+// Input: {"job_id": 123, "field1": "hello", "field2": "world"}, Fields: ["field1", "field2"], Target: "data", Result: {"job_id": 123, "data": {"field1": "hello", "field2": "world"}}
+// Input: {"job_id": 123, "field1": "hello", "field2": "world"}, Fields: ["!job_id"], Target: "data", Result: {"job_id": 123, "data": {"field1": "hello", "field2": "world"}}
+func (p *Move) ApplyMove(v dyn.Value) (dyn.Value, error) {
+	mapping, ok := v.AsMap()
+	if !ok {
+		return dyn.InvalidValue, fmt.Errorf("expected a map, but found %s", v.Kind())
+	}
+
+	// Create a new mapping for the target field
+	targetMapping := dyn.NewMapping()
+
+	// Create a new result mapping
+	resultMapping := dyn.NewMapping()
+
+	// Check if we have any negated fields (starting with !)
+	hasNegatedFields := false
+	var negatedFields []string
```

Comment:
Shall we raise the error if we use both negated and normal keys? Otherwise behaviour is confusing, I guess normal fields are skipped?